mods/README.md

```diff
@@ -77,7 +77,7 @@ The list should be structured in the following way:
 
 - Mounts the Triangle Labs 2-to-1 adapter directly to the mosquito toolhead.
 
-[Tablet mount](../mods/Danowar/Tablet_Mount)
+[Tablet mount](../Mods/Danowar/Tablet_Mount)
```

Comment:
The folder path will not be correct and breaks this link in README.md

Reply:
Yep, "mods" is now lower-case.